### DISCUSSION TITLE
Appkit v4? or v5?

### DISCUSSION THREAD

**Discussion by soharang** [WalletConnect/walletconnect-monorepo#5180] [Developer Support]
I have checked the following page: https://docs.walletconnect.com/appkit/upgrade#web3modal-v2-to-appkit-v5-upgrade-guide

The title of the page is "Web3Modal v2 to AppKit v5 Upgrade Guide" and the head message says: "This guide is useful for those that have used a previous Web3Modal V2 version and are looking to upgrade to AppKit v5."

However, it seems that the guide is for AppKit v4, as the content mentions AppKit v4. Specifically, it says "To upgrade from Web3Modal v2 to AppKit v4, start by removing Web3Modal v2 dependencies" in the installation section.

Could you confirm which one is correct - upgrading to AppKit v4 or v5?

**Comment by glitch-txs** [comment]
There are no breaking changes between v4 and v5 besides the enableEmail flag and the email connector. So it's basically the same setup :)

**Comment by glitch-txs** [comment]
Seems like we need to update the documentation

**Comment by soharang** [comment]
Thanks for the confirmation! 
Now I understand how it works, and it would be great if the documentation used a single version number - v5. :D
